openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -1146,6 +1157,9 @@
       "type": "string",
       "minLength": 1,
       "maxLength": 32
+    },
+    "changeSummary": {
+      "$ref": "../../type/changeSummaryMap.json"
     }
```

Comment:
maybe 2 last thoughts on this:
- would it make sense for the `changeSummary` to be inside `changeDescription` as another field? Just to centralize all the change stuff. Unless we have it outside for another reason
- if the default is null, do we really need the migration adding an empty JSON 🤔